examples/core/transform-feedback-instanced/app.js

```diff
@@ -271,25 +234,34 @@ const animationLoop = new AnimationLoop({
     gl,
     width,
     height,
-    vertexArrays,
-    transformVertexArrays,
-    transformFeedbacks,
     modelRender,
-    modelTransform
+    positionBuffer,
+    colorBuffer,
+    bufferMap
   }) {
 
     gl.viewport(0, 0, width, height);
     gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
     const destinationIdx = (currentSourceIdx + 1) % 2;
 
-    modelTransform.draw({
-      vertexArray: transformVertexArrays[currentSourceIdx],
-      transformFeedback: transformFeedbacks[destinationIdx]
-    });
+    bufferMap.run();
+
+    const offsetBuffer = bufferMap.getBuffer('v_offset');
+    const rotationBuffer = bufferMap.getBuffer('v_rotation');
 
+    bufferMap.swapBuffers();
+
+    offsetBuffer.updateLayout({instanced: 1});
```

Comment:
opportunity for further improvement:

Having to do these layout updates really complicates the API. I think there is a way to avoid this, let's discuss.